feedthefox/settings.py

```diff
@@ -98,6 +99,7 @@
 
 MEDIA_ROOT = config('MEDIA_ROOT', default=os.path.join(BASE_DIR, 'media'))
 MEDIA_URL = config('MEDIA_URL', '/media/')
+USER_PHOTOS_DIR = config('USER_PHOTOS_DIR', 'uploads/profiles')
```

Comment:
That was my intention! I just used it in .env and didn't append it as default option... Sorry about that!